app/assets/javascripts/components/faye_messages.jsx

```diff
@@ -0,0 +1,41 @@
+class FayeMessages extends React.Component {
+  constructor(props) {
+    super(props);
+     this.state = {
+      urlType:`/assets/chatroom.jpg`
+      };
+  }
+
+  generate_private_chat(e) {
```

Comment:
use event as attribute name not e and function name should be camelCase not generate_private_chat but generatePrivateChat